### DISCUSSION TITLE
iFrame shared URL

### DISCUSSION THREAD

**Discussion by Jamesking56** [plausible/analytics#815] [Anything Else]
Hi,

We're an agency looking to integrate Plausible into our custom built client area. The advantages for our clients mean that we can manage their analytics (and their website with us) via a single login for them to remember.

We have been looking into using the new Stats API to do this integration, however, we really just want to replicate exactly what is shown on the site dashboard.

So, with this in mind, an even easier solution for us would be if we could simply iFrame embed the client's privately shared URL for their site from Plausible, so they can use Plausible's dashboard inside of our client area.

It looks like, however, Plausible's dashboards currently block iFrames meaning this is not possible.

So, can Plausible just disable iFrame protection a shared URL? This would massively simplify our integration.

**Comment by metmarkosaric** [comment]
thanks @Jamesking56! we're in the process of testing the embed mode for our dashboard #213 so it will not take that long before we have this ready. it will work exactly the same way as you're describing by embedding shared link dashboards.

**Comment by metmarkosaric** [comment]
Embed mode is now ready. See the docs: https://plausible.io/docs/embed-dashboard

**Reply by NinjutsuSunrise** [reply]
This is awesome, thank you! We're currently trying to solve for a similar scenario as OP... but based on Custom Properties to segment the analytics. Small blocker on our end being that the end user can remove / adjust filters and download exports. Posted an idea to help mitigate that here https://github.com/plausible/analytics/discussions/1495#discussioncomment-6242884
